xbmc/video/dialogs/GUIDialogSubtitles.cpp

```diff
@@ -380,6 +402,51 @@ void CGUIDialogSubtitles::OnSearchComplete(const CFileItemList *items)
   SetInvalid();
 }
 
+void CGUIDialogSubtitles::OnSubtitleServiceContextMenu(const int itemIdx)
```

Comment:
it is the same (unwritten?) rule that applies to return values: never return from/pass in a plain const. it‘s either 1. `T` like here because it‘s an `int` not a heavy object, or 2. `T&`, or 3. `const T&`

Reply:
Right.